### PR TITLE
Update JS samples to use latest API and add error handling

### PR DESCRIPTION
**Pull request by aiham**
Also make JS style more uniform and fix URL for JS SDK.

### CODE REVIEW

File: sample/Archiving/public/js/host.js [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
+var session = OT.initSession(apiKey, sessionId),
     publisher = OT.initPublisher('publisher'),
     archiveID = null;
 
-session.connect(apiKey, token, function(err, info) {
-  if(err) {
-    alert(err.message || err);
+session.connect(token, function(error, info) {
+  if (error) {
+    console.error('Failed to connect', error);
```
Comment (Lucashuang0802) [inline sample/Archiving/public/js/host.js]: Should we use `console.error` or `alert`(which I used from the last PR)? If later, I will adjust later

File: sample/HelloWorld/public/js/helloworld.js [hunk 2]
```diff
@@ -6,8 +6,6 @@ var publisher = OT.initPublisher('publisher', {
     insertMode: 'append',
     width: '100%',
     height: '100%'
-}, function(error) {
-  alert(error);
 });
```
Comment (Lucashuang0802) [inline sample/HelloWorld/public/js/helloworld.js]: Wouldn't that be interesting to keep this callback? I feel like a lot of people do have camera/microphone permission issues.

**Comment by aiham** [comment]
@Lucashuang0802 Okay, I'll return the `initPublisher` callback but `console.error` is preferred over `alert` in modern web development.

**Comment by Lucashuang0802** [comment]
👍 LGTM

**Pull request merged**